slacker.py

```diff
@@ -20,20 +26,53 @@ def __init__(self):
         self.logger.setLevel(logging.INFO)
         self.logger.addHandler(logging.StreamHandler(sys.stdout))
 
-        if "SLACK_API_TOKEN" not in os.environ:
-            raise Exception("SLACK_API_TOKEN not provided in env variables")
+        if "SLACK_USER_TOKEN" not in os.environ:
+            raise Exception("SLACK_USER_TOKEN not provided in env variables")
+        if "SLACK_BOT_TOKEN" not in os.environ:
+            raise Exception("SLACK_BOT_TOKEN not provided in env variables")
```

Comment:
Get variables from os.environ isn't a task of slacker, because Slacker is our client for SlackAPI and this class should get existing variables.

Reply:
fixed